vendor/wp-now/src/wp-now.ts

```diff
@@ -433,10 +435,15 @@ async function initWordPress(
 		initializeDefaultDatabase = true;
 	}
 
-	const wpConfigConsts = {
-		WP_HOME: siteUrl,
-		WP_SITEURL: siteUrl,
-	};
+	const wpConfigConsts = {};
+
+	if ( ! wpConfigConstants?.WP_SITEURL ) {
+		wpConfigConsts[ 'WP_SITEURL' ] = siteUrl;
+	}
+
+	if ( ! wpConfigConstants?.WP_HOME ) {
+		wpConfigConsts[ 'WP_HOME' ] = siteUrl;
+	}
```

Comment:
We set these constants only if they are not defined in wp-config.php.
If they are defined, then we do nothing and regular WordPress will use them.

Reply:
When tested https://github.com/Automattic/studio/pull/936/commits/e98835dfe9bbda9b30039eb4120c80d778295d5e it seems setting the siteUrl was needed. After testing it now, I can create new sites without saving the siteurl in constants.